packages/lesswrong/components/posts/PostsPage/PostsPageEventData.tsx

```diff
@@ -22,11 +22,8 @@ const styles = (theme: ThemeType): JssStyles => ({
     display: 'flex',
     justifyContent: 'space-between',
     marginTop: theme.spacing.unit*2,
-    ...theme.typography.postStyle,
```

Comment:
should this be forum-gated?

Reply:
I can't remember why I thought that was necessary, but just removing it now.

Reply:
I meant the removal of the style - it exists in master.  Is it not doing anything?

Reply:
Oh, nevermind, you put it back on a different line.